dask/dataframe/io/parquet/arrow.py

```diff
@@ -762,6 +766,28 @@ def write_metadata(cls, parts, meta, fs, path, append=False, **kwargs):
     # Private Class Methods
     #
 
+    @classmethod
+    def _update_open_file_func(cls, fs, storage_options, kwargs):
+        """Update kwargs if we can use a native pyarrow filesystem
+
+        Currently supports ``s3fs`` -> ``pyarrow.fs.S3FileSystem``.
+        """
+        is_s3fs = type(fs).__module__.split(".")[0] == "s3fs"
```

Comment:
As is, I think we could use `dask.utils.typename(...)` here to make things more succinct. That said, I wonder if we could use a more direct check using `isinstance(fs, fsspec.AbstractFileSystem)` and `fs.protocol` (or something along these lines)

Reply:
Yeah, this line is pretty gross.  If we cant boil this down to a direct instance check, then I like the idea of using a well-defined utility.